src/System.Net.Security/tests/StressTests/SslStress/Configuration.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net;
+
+namespace SslStress
+{
+    [Flags]
+    public enum RunMode { server = 1, client = 2, both = server | client };
```

Comment:
`server` etc. don't conform to naming. Is this because the command line parser is case sensitive?

Reply:
That's correct.

Reply:
Can you add a comment to that effect?  Otherwise, it looks so strange to have these be lower case, I fully expect someone will come along and "fix" it ;)